src/NuGetForUnity.Tests/Assets/Tests/Editor/NuGetTests.cs

```diff
@@ -775,6 +775,37 @@ public void TestSlimRestoreInstall(string packageId, string packageVersion)
             package.Version);
     }
 
+    [Test]
+    [TestCase("Microsoft.Azure.WebJobs.Sources", "3.0.37")]
+    [TestCase("NServiceBus.Testing.Fakes.Sources", "7.1.13")]
+    public void TestSourceCodePackageInstall(string packageId, string packageVersion)
```

Comment:
I added a check to see if `Sources` directory is added in the package root and ensured that `contentFiles` directory doesn't exist. I'm trying to thing of other generic checks that we could do while installing this but nothing comes to mind as directories/files under `Sources` will be very package-specific. H